music21/stream/makeNotation.py

```diff
@@ -522,6 +522,7 @@ def makeMeasures(
             v = stream.Voice()
             v.id = voiceIndex  # id is voice index, starting at 0
             m.coreInsert(0, v)
+        m.coreElementsChanged()
```

Comment:
`if(voiceCount): m.coreElementsChanged()` to avoid calling in the normal case where there are no voices. (which is why I didn't discover this bug before)